packages/bot-web-ui/src/components/transactions/transactions.jsx

```diff
@@ -57,18 +57,17 @@ const Transactions = observer(({ is_drawer_open }) => {
             })}
         >
             <div className='download__container transaction-details__button-container'>
-                <DesktopWrapper>
-                    <Download tab='transactions' />
-                </DesktopWrapper>
+                <Download tab='transactions' />
                 <Button
-                    id='download__container-view-detail-button'
-                    className='download__container-view-detail-button'
+                    id='view-detail-button'
+                    className='view-detail-button'
```

Comment:
why not following the BEM structure for the naming as we had it on the prev version?

Reply:
Done